playpen/repo_level_awareness/codeplan.py

```diff
@@ -86,6 +86,9 @@ def execute_task(self, task: Task) -> TaskResult:
 
     def get_agent_for_task(self, task: Task) -> Agent:
         for agent in self.agents:
+            print(agent)
+            print(task)
+            print("here!!!!")
```

Comment:
Nit: Unsure if you meant to leave this here

Reply:
updated